atlasdb-commons/src/main/java/com/palantir/exception/NotInitializedException.java

```diff
@@ -16,18 +16,40 @@
 package com.palantir.exception;
 
 import com.palantir.common.exception.AtlasDbDependencyException;
+import com.palantir.logsafe.Arg;
+import com.palantir.logsafe.Safe;
 import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.SafeLoggable;
+
+import java.util.List;
+
+public class NotInitializedException extends AtlasDbDependencyException implements SafeLoggable {
```

Comment:
https://github.com/palantir/atlasdb/pull/4369/files we did previously, which should make this OK from an ABI break perspective. Also constructor args are the same, and I'd imagine `SafeLoggable` does an `instanceof` call, which hopefully is determined at runtime?